server/src/main/java/org/elasticsearch/index/mapper/BlockLoader.java

```diff
@@ -501,4 +503,10 @@ interface SingletonOrdinalsBuilder extends Builder {
          */
         SingletonOrdinalsBuilder appendOrd(int value);
     }
+
+    interface AggregateMetricDoubleBuilder extends Builder {
```

Comment:
Interesting! This is not something in `server` but we have to add it here for this. Huh. I suppose that's ok.